bin/index.js

```diff
@@ -145,7 +160,28 @@ const handleMainPrompts = (repo, token, ans) => {
         validate: validateAddPackages
       }])
       .then((ans)=>{
-        return addFromPackage( repo, token, ans.path );
+        if (ans.path !== "") {
+          return addFromPackage( repo, token, ans.path );
+        } else {
+          try {
+            addFromPackage( repo, token, getGlobalPackage() );
+          } catch (e) {
+            console.log("Either an incorrect global was used or no global package has been set yet.");
+          }
+        }
+      })
+      .catch(console.warn);
+      break;
+
+    case "add global package":
+      prompt([{
```

Comment:
You can break off this array for the prompt and put it in the [`./bin/prompts/`](https://github.com/himynameisdave/git-labelmaker/tree/v0.9.0/bin/prompts) directory 👍